src/ServerMainComponent.cpp

```diff
@@ -18,8 +18,10 @@
 #elif JUCE_LINUX
 #include "isobus/hardware_integration/socket_can_interface.hpp"
 #endif
+#include "isobus/isobus/can_stack_logger.hpp"
 
 #include <chrono>
+#include <filesystem>
```

Comment:
I still have my doubts that filesystem will work well on android or ios or whatever as opposed to the juce `File`